cogs/timer.py

```diff
@@ -2270,6 +2432,20 @@ async def duringTimer(self,ctx, datestart, startTime, startTimes, role, game, au
                 elif (msg.content.startswith(f"{commandPrefix}timer reward") or msg.content.startswith(f"{commandPrefix}t reward")):
                     if await self.permissionCheck(msg, author):
                         startTimes,dmChar = await self.reward(ctx, msg=msg, start=startTimes,dmChar=dmChar)
+                elif (msg.content.startswith(f"{commandPrefix}timer major") or msg.content.startswith(f"{commandPrefix}t major")): #Random Major
+                    if await self.permissionCheck(msg, author):
+                        rewardItem = None
+                        while rewardItem == None: # This repeats if a duplicate is found.
+                            rewardItem = await self.randomRew(ctx, msg=msg, start=startTimes,dmChar=dmChar, rewardType="Major", amount=1)
```

Comment:
I thought the goal was to allow multiple?

Reply:
For the out-of-timer command.